tests/unit/test_run_logprep.py

```diff
@@ -57,12 +57,12 @@ def test_raises_getter_error_for_not_existing_protocol(self):
         with pytest.raises(GetterNotFoundError, match="No getter for protocol 'almighty_protocol'"):
             run_logprep.main()
 
-    @mock.patch("requests.get")
-    def test_gets_config_from_https(self, mock_request):
+    @responses.activate
+    def test_gets_config_from_https(self):
         pipeline_config = Path("quickstart/exampledata/config/pipeline.yml").read_text(
             encoding="utf8"
         )
-        mock_request.return_value.text = pipeline_config
+        responses.add(responses.GET, "https://does.not.exits/pipline.yml", pipeline_config)
```

Comment:
Even though this is just a test, can we change the following, please?

```suggestion
        responses.add(responses.GET, "https://does.not.exist/pipeline.yml", pipeline_config)
```

Reply:
This also needs to be changed below in order to be consistent in this test, of course. (l. 70)